boards/shields/max3421e/doc/index.rst

```diff
@@ -0,0 +1,58 @@
+.. _max3421e_shield:
+
+MAX3421E Shield
+###############
+
+Overview
+********
+
+Support for expansion boards (shields) based on MAX3421E USB Peripheral/Host
+controller.
+
+Sparkfun USB Host Shield
+************************
+
+Pins Assignment of the Shield Connector
+=======================================
+
++-----------------------+-----------------------+-----------------------------+
+| Shield Connector Pin  | Function              | MAX3421E Pin                |
++=======================+=======================+=============================+
+| D0                    | not connected         |                             |
++-----------------------+-----------------------+-----------------------------+
+| D1                    | not connected         |                             |
++-----------------------+-----------------------+-----------------------------+
+| D2                    | not connected         |                             |
++-----------------------+-----------------------+-----------------------------+
+| D3                    | not connected         |                             |
++-----------------------+-----------------------+-----------------------------+
+| D4                    | not connected         |                             |
++-----------------------+-----------------------+-----------------------------+
+| D5                    | not connected         |                             |
++-----------------------+-----------------------+-----------------------------+
+| D6                    | not connected         |                             |
++-----------------------+-----------------------+-----------------------------+
+| D7                    | RESET pin             |                             |
++-----------------------+-----------------------+-----------------------------+
+| D8                    | not connected         |                             |
++-----------------------+-----------------------+-----------------------------+
+| D9                    | INTn                  | 24 (Interrupt Output)       |
++-----------------------+-----------------------+-----------------------------+
+| D10                   | SPI CSn               | 34 (SPI Chip Select)        |
++-----------------------+-----------------------+-----------------------------+
+| D11                   | SPI MOSI              | 36 (SPI Serial Data Input)  |
++-----------------------+-----------------------+-----------------------------+
+| D12                   | SPI MISO              | 35 (SPI Serial Data Out)    |
++-----------------------+-----------------------+-----------------------------+
+| D13                   | SPI SCK               | 37 (SPI Serial Clock Input) |
++-----------------------+-----------------------+-----------------------------+
```

Comment:
Where are the MAX3421E pin numbers from? The [Sparkfun host shield datasheet](https://www.sparkfun.com/datasheets/DevTools/Arduino/USBHostShield-v13.pdf) has different numbers in it for the pins.

Reply:
Where do you see differences? These are shield pins not MAX3421E pin numbers.

Reply:
I see. The third column is labeled "MAX3421E Pin", so I looked at the pins on the MAX.